Dockerfile

```diff
@@ -14,18 +14,29 @@ COPY package.json .
 COPY yarn.lock .
 
 RUN yarn install --prod
+RUN cp -R node_modules /usr/app/prod_node_modules
```

Comment:
I would recommend doing `yarn install` (all dependencies) in this `dependencies` layer, then not copying `node_modules` and instead running `yarn install --prod` in the runner image to simplify it a little

Reply:
I've done it avoid downloading prod dependencies twice. That way it seems to work faster. But if you prefer calling yarn install twice I will change it.